cirq-google/cirq_google/json_test_data/cirq.google.EngineResult.repr

```diff
@@ -1 +1 @@
-cirq_google.EngineResult(params=cirq.ParamResolver({sympy.Symbol('a'): 0.5}), records={'m': np.array([[[True, True, False, True, False]], [[False, True, True, False, False]], [[True, False, True, False, True]]], dtype=np.bool)}, job_id='my_job_id', job_finished_time=datetime.datetime(2022, 4, 1, 8, 23, 45, tzinfo=datetime.timezone.utc))
+cirq_google.EngineResult(params=cirq.ParamResolver({sympy.Symbol('a'): 0.5}), records={'m': np.array([[[True, True, False, True, False]], [[False, True, True, False, False]], [[True, False, True, False, True]]], dtype=bool)}, job_id='my_job_id', job_finished_time=datetime.datetime(2022, 4, 1, 8, 23, 45, tzinfo=datetime.timezone.utc))
```

Comment:
This `repr` is not what gets generated if I call `repr(engine)` where `engine` is the corresponding object, right? 

For example, this is what I get when I run `repr(engine)` using your PR, where `dtype=np.dtype('bool')` instead of `dtype=bool`. 

```python
>>> repr(engine)
cirq_google.EngineResult(params=cirq.ParamResolver({sympy.Symbol('a'): 0.5}), records={'m': np.array([[[True, True, False, True, False]], [[False, True, True, False, False]], [[True, False, True, False, True]]], dtype=np.dtype('bool'))}, job_id='my_job_id', job_finished_time=datetime.datetime(2022, 4, 1, 8, 23, 45, tzinfo=datetime.timezone.utc))
```


In general, all the `.repr` files should contain the output of calling `repr(val)` for each `val`. Since we are changing the `repr` generation, we'd need to update every object that is impacted by this change. i.e. Let `Object` be a class s.t. `repr(object)` used to contain `np.int` and now contains `np.dtype(int)`. We need to ensure that `Object.repr` is updated to reflect the new output of `repr(object)`.